packages/runtime/test/fixtures/escrow-account/assets/escrow.teal

```diff
@@ -15,10 +15,6 @@ txn RekeyTo
 global ZeroAddress
 ==
 &&
-txn CloseRemainderTo
-global ZeroAddress
-==
-&&
```

Comment:
why are we removing these lines ?

Reply:
I added a test for `closeRemainderTo` here. So removed these lines. If we update it for a specific address, then in each test i'll have to specify `closeRemainderTo`, which we don't want

Reply:
ok